ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml

```diff
@@ -36,8 +36,10 @@ tests:
       commands: |
         if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:index-v1.3.x;
+          export RUN_TYPE='scratch';
         else
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index;
+          export RUN_TYPE='pr-check';
```

Comment:
@tisutisu The job is missing the part for uploading the KPI to opensearch

Reply:
My Bad. Added the uploading data part now.